src/client.cpp

```diff
@@ -331,6 +331,20 @@ namespace quicr {
                 sub_it->second.get()->SetStatus(SubscribeTrackHandler::Status::kNotSubscribed);
                 return true;
             }
+            case messages::ControlMessageType::kSubscribesBlocked: {
```

Comment:
This needs to be added to server as well.  Both could receive subscribes blocked, although it's likely that only the server will be the one that actually receives this.